### PR TITLE
Add radiative bc where the emissivity function is supplied by the user

### PR DESCRIPTION
**Pull request by jasondhales**
See #13053

### CODE REVIEW

File: modules/heat_conduction/test/tests/radiative_bcs/tests [hunk 1]
```diff
@@ -7,4 +7,13 @@
     design = 'source/bcs/InfiniteCylinderRadiativeBC.md'
     issues = "#13053"
   [../]
+  [./function_radiative_bc]
+    type = Exodiff
+    input = 'function_radiative_bc.i'
+    exodiff = 'function_radiative_bc_out.e'
+    requirement = "MOOSE shall be able to model radiative heat transfer using a user-specified emissivity function."
+    design = 'source/bcs/FunctionRadiativeBC.md'
+    issues = "#13053"
```
Comment (loganharbour) [inline modules/heat_conduction/test/tests/radiative_bcs/tests:16]:
```suggestion
    issues = "#13053 #16962"
```

File: modules/heat_conduction/test/tests/radiative_bcs/tests [hunk 2]
```diff
@@ -7,4 +7,13 @@
...
+    mesh_mode = REPLICATED
```
Comment (loganharbour) [inline modules/heat_conduction/test/tests/radiative_bcs/tests:17]: What causes the restriction for replicated mesh?

**Review by loganharbour — CHANGES_REQUESTED** [review]

File: modules/heat_conduction/test/tests/radiative_bcs/tests (hunk 2 above)
Comment (jasondhales) [reply modules/heat_conduction/test/tests/radiative_bcs/tests:17]: The tests died without it.  The mesh has two independent chunks with a unique sideset on each chunk.  I assumed that distributed mesh couldn't handle that situation and added REPLICATED to the tests file.

File: modules/heat_conduction/test/tests/radiative_bcs/tests (hunk 1 above)
Comment (jasondhales) [reply modules/heat_conduction/test/tests/radiative_bcs/tests:16]: I'm curious:  are we supposed to put PR ids in all new issues lines?  I thought they were for Issues only.

Comment (loganharbour) [reply modules/heat_conduction/test/tests/radiative_bcs/tests]: Crud. This was what happens when I review late in the evening... you're absolutely correct.

**Review by loganharbour — CHANGES_REQUESTED** [review]
Revert my silly mistake for the issue reference and we're good.

Sorry about that.

**Comment by jasondhales** [comment]
> Revert my silly mistake for the issue reference and we're good.
> 
> Sorry about that.

No problem!  Done!

**Review by loganharbour — APPROVED** [review]
Good to merge once next is green again.

**Pull request merged**